packages/ts-client/src/webrtc/webRTCEndpoint.ts

```diff
@@ -574,7 +572,7 @@ export class WebRTCEndpoint<EndpointMetadata = any, TrackMetadata = any> extends
     const resolutionNotifier = new Deferred<void>();
 
     this.commandsQueue.pushCommand({
-      handler: () => {
+      handler: async () => {
         this.localTrackManager.removeTrackHandler(trackId);
       },
```

Comment:
```suggestion
      handler: async () => this.localTrackManager.removeTrackHandler(trackId),
```
So if removeTrackHandler will be async in future, it will work correctly